armi/reactor/components/component.py

```diff
@@ -1237,7 +1238,8 @@ def getReactionRateDict(nucName, lib, xsType, mgFlux, nDens):
     assume there is no n3n cross section in ISOTXS
 
     """
-    key = "{}{}A".format(nucName, xsType)
+    nucLabel = nuclideBases.byName[nucName].label
+    key = "{}{}A".format(nucLabel, xsType)
```

Comment:
Based on the comment above, this "key" would be replaced with `f"{nucLabel}{microSuffix}"`